libs/barista-components/chart/src/heatfield/chart-heatfield.ts

```diff
@@ -228,8 +228,11 @@ export class DtChartHeatfield extends _DtHeatfieldMixinBase
    * @internal
    */
   _handleKeydown(event: KeyboardEvent): void {
-    if (_readKeyCode(event) === ENTER) {
-      this._toggleActive();
+    if (_readKeyCode(event) === ESCAPE) {
+      this.active = false;
+    }
```

Comment:
```suggestion
```
I think once would be enough, @ffriedl89 I guess this happend with your rebase conflict, maybe you can take care of that one as well.